### PR TITLE
[Feature](Compaction)Support full compaction

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

When MoW is enabled, there may be a calculation error of the delete bitmap, resulting in data errors. This full compaction function will perform compaction on all tablets and recalculate the delete bitmap to restore the data.

Usage:
```bash
curl -X POST "http://be_host:be_http_port/api/compaction/run?tablet_id=xxx&compact_type=full"    
e.g. curl -X POST "http://127.0.0.1:8040/api/compaction/run?tablet_id=10084&compact_type=full"   
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.82 seconds
 stream load tsv:          459 seconds loaded 74807831229 Bytes, about 155 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          69.4 seconds inserted 10000000 Rows, about 144K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230629110105_clickbench_pr_169850.html

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 40.74 seconds
 stream load tsv:          453 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          66.4 seconds inserted 10000000 Rows, about 150K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230629140302_clickbench_pr_169982.html

File: be/src/olap/full_compaction.cpp [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    RETURN_IF_ERROR(_check_rowset_overlapping(_input_rowsets));
+    if (_input_rowsets.size() <= 1) {
+        return Status::Error<BE_NO_SUITABLE_VERSION>();
+    }
+
+    // If there are delete predicate rowsets in tablet, start_version > 0 implies some rowsets before
+    // delete version cannot apply these delete predicates, which can cause incorrect query result.
+    // So we must abort this base compaction.
+    // A typical scenario is that some rowsets before cumulative point are on remote storage.
+    if (_input_rowsets.front()->start_version() > 0) {
```
Comment (zhannngchen) [inline be/src/olap/full_compaction.cpp:96]: 不需要检查这个，这里只需要检查是否包含了所有的rowset就行
这个检查是宇轩加的，为了防止有一些冷数据存到了S3上，无法参与base compaction而做的处理
但是full compaction的目的就是为了compact所有的rowset

File: be/src/olap/full_compaction.cpp [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    // 1. pick rowsets to compact
+    RETURN_IF_ERROR(pick_rowsets_to_compact());
+    TRACE_COUNTER_INCREMENT("input_rowsets_count", _input_rowsets.size());
+    _tablet->set_clone_occurred(false);
+
+    return Status::OK();
+}
+
+Status FullCompaction::execute_compact_impl() {
+    std::unique_lock<std::mutex> lock(_tablet->get_full_compaction_lock(), std::try_to_lock);
```
Comment (zhannngchen) [inline be/src/olap/full_compaction.cpp:59]: same as above

File: be/src/http/action/compaction_action.h [hunk 3]
```diff
@@ -39,6 +39,7 @@ enum class CompactionActionType {
 const std::string PARAM_COMPACTION_TYPE = "compact_type";
 const std::string PARAM_COMPACTION_BASE = "base";
 const std::string PARAM_COMPACTION_CUMULATIVE = "cumulative";
+const std::string PARAM_COMPACTION_FULL = "full";
```
Comment (zhannngchen) [inline be/src/http/action/compaction_action.h:42]: You need to process this new compaction type in `TabletReader::_init_delete_condition`

File: regression-test/suites/compaction/test_full_compaction.groovy [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+
+        // version5 (1,100)(2,200)(3,100)
+        sql """update ${tableName} set value = 100 where user_id = 3"""
+        qt_5 """select * from ${tableName}"""
+
+
+        // version6 (1,100)(2,200)
+        sql """delete from ${tableName} where user_id = 3"""
+        qt_6 """select * from ${tableName}"""
```
Comment (zhannngchen) [inline regression-test/suites/compaction/test_full_compaction.groovy:91]: add sequence column and delete sign, to make sure they all works correctly

File: be/src/olap/full_compaction.cpp [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    }
+
+    SCOPED_ATTACH_TASK(_mem_tracker);
+
+    // 2. do base compaction, merge rowsets
+    int64_t permits = get_compaction_permits();
+    RETURN_IF_ERROR(do_compaction(permits));
+
+    // 3. set state to success
+    _state = CompactionState::SUCCESS;
```
Comment (zhannngchen) [inline be/src/olap/full_compaction.cpp:79]: you should also consider to update cumulative point

**Comment by hello-stephen** [comment]
(From new mechine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.1 seconds
 stream load tsv:          508 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          88.7 seconds inserted 10000000 Rows, about 112K ops/s
 storage size: 17167586821 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230706003429_clickbench_pr_172997.html

File: be/src/olap/compaction.cpp [hunk 6]
```diff
@@ -690,6 +690,22 @@ Status Compaction::check_version_continuity(const std::vector<RowsetSharedPtr>&
     return Status::OK();
 }
 
+Status Compaction::check_all_version(const std::vector<RowsetSharedPtr>& rowsets) {
+    if (rowsets.empty()) {
+        return Status::Error<FULL_MISS_VERSION>();
+    }
+    const RowsetSharedPtr& last_rowset = rowsets.back();
```
Comment (zhannngchen) [inline be/src/olap/compaction.cpp:697]: Check the 1st rowset's start version is 0 or 2

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.22 seconds
 stream load tsv:          455 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          67.0 seconds inserted 10000000 Rows, about 149K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230706052319_clickbench_pr_173335.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.34 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          87.9 seconds inserted 10000000 Rows, about 113K ops/s
 storage size: 17164164369 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230706134009_clickbench_pr_173337.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.39 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          89.6 seconds inserted 10000000 Rows, about 111K ops/s
 storage size: 17170482229 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230709025914_clickbench_pr_174977.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 49.48 seconds
 stream load tsv:          499 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.6 seconds inserted 10000000 Rows, about 349K ops/s
 storage size: 17167837491 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230709232221_clickbench_pr_175287.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 55.7 seconds
 stream load tsv:          498 seconds loaded 74807831229 Bytes, about 143 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17162507468 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230710012536_clickbench_pr_175326.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.24 seconds
 stream load tsv:          499 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.5 seconds inserted 10000000 Rows, about 350K ops/s
 storage size: 17163424328 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230710121821_clickbench_pr_175653.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 56.99 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.7 seconds inserted 10000000 Rows, about 348K ops/s
 storage size: 17167114567 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230710154710_clickbench_pr_175757.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.05 seconds
 stream load tsv:          497 seconds loaded 74807831229 Bytes, about 143 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.7 seconds inserted 10000000 Rows, about 348K ops/s
 storage size: 17164027644 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230711114637_clickbench_pr_176188.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 49.45 seconds
 stream load tsv:          506 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         18 seconds loaded 2358488459 Bytes, about 124 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.3 seconds inserted 10000000 Rows, about 353K ops/s
 storage size: 17161507410 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712121729_clickbench_pr_176867.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.72 seconds
 stream load tsv:          501 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          28.8 seconds inserted 10000000 Rows, about 347K ops/s
 storage size: 17166492685 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712123007_clickbench_pr_176885.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.36 seconds
 stream load tsv:          501 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.4 seconds inserted 10000000 Rows, about 352K ops/s
 storage size: 17170131222 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712153211_clickbench_pr_176975.html

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.41 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          28.5 seconds inserted 10000000 Rows, about 350K ops/s
 storage size: 17164273606 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712162441_clickbench_pr_176985.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.47 seconds
 stream load tsv:          449 seconds loaded 74807831229 Bytes, about 158 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          25.9 seconds inserted 10000000 Rows, about 386K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712091203_clickbench_pr_177023.html

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.92 seconds
 stream load tsv:          502 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         18 seconds loaded 2358488459 Bytes, about 124 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          30.0 seconds inserted 10000000 Rows, about 333K ops/s
 storage size: 17160414738 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712190210_clickbench_pr_177157.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.27 seconds
 stream load tsv:          502 seconds loaded 74807831229 Bytes, about 142 MB/s
 stream load json:         18 seconds loaded 2358488459 Bytes, about 124 MB/s
 stream load orc:          66 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17167488571 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712190229_clickbench_pr_177184.html

File: be/src/olap/compaction.cpp [hunk 7]
```diff
@@ -716,6 +720,25 @@ Status Compaction::check_version_continuity(const std::vector<RowsetSharedPtr>&
     return Status::OK();
 }
 
+Status Compaction::check_all_version(const std::vector<RowsetSharedPtr>& rowsets) {
```
Comment (zhannngchen) [inline be/src/olap/compaction.cpp:723]: Why add such a method in base class? It's only used in FullCompaction

File: be/src/olap/tablet.cpp [hunk 8]
```diff
@@ -3248,6 +3266,82 @@ Status Tablet::commit_phase_update_delete_bitmap(
     return Status::OK();
 }
 
+Status Tablet::full_compaction_update_delete_bitmap(const RowsetSharedPtr& rowset,
```
Comment (zhannngchen) [inline be/src/olap/tablet.cpp:3269]: Is it possible to move these methods to FullCompaction class? Too much xx_update_delete_bitmap in Tablet class

File: be/src/olap/full_compaction.cpp [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+        // the tablet is with rowset: [0-1], [2-y]
+        // and [0-1] has no data. in this situation, no need to do full compaction.
+        return Status::Error<FULL_NO_SUITABLE_VERSION>("There is no suitable version");
+    }
+
+    return Status::OK();
+}
+
+Status FullCompaction::modify_rowsets(const Merger::Statistics* stats) {
+    RETURN_IF_ERROR(
```
Comment (zhannngchen) [inline be/src/olap/full_compaction.cpp:110]: Add the check here, only MoW table needs to update delete bitmap
```
 if (_tablet->keys_type() == KeysType::UNIQUE_KEYS &&
        _tablet->enable_unique_key_merge_on_write()) {
```

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.25 seconds
 stream load tsv:          509 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17167658146 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230713153756_clickbench_pr_177918.html

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.86 seconds
 stream load tsv:          508 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.8 seconds inserted 10000000 Rows, about 347K ops/s
 storage size: 17166433719 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230713172657_clickbench_pr_178037.html

File: be/src/olap/full_compaction.cpp [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+    }
+
+    for (const auto& it : tmp_rowsets) {
+        const int64_t& cur_version = it->rowset_meta()->start_version();
+        RETURN_IF_ERROR(
+                _full_compaction_calc_delete_bitmap(it, rowset, cur_version, rowset_writer));
+    }
+
+    std::lock_guard rowset_update_lock(_tablet->get_rowset_update_lock());
+    std::lock_guard header_lock(_tablet->get_header_lock());
```
Comment (zhannngchen) [inline be/src/olap/full_compaction.cpp:171]: you should hold these locks before L157

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 52.52 seconds
 stream load tsv:          510 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.8 seconds inserted 10000000 Rows, about 347K ops/s
 storage size: 17169102459 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230714110848_clickbench_pr_178294.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.84 seconds
 stream load tsv:          512 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.8 seconds inserted 10000000 Rows, about 347K ops/s
 storage size: 17168626843 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230714110928_clickbench_pr_178302.html

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 56.73 seconds
 stream load tsv:          513 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17166997714 Bytes

**Review by zhannngchen — APPROVED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run p0

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**